test/users.js

```diff
@@ -695,5 +695,198 @@ describe('/users', function () {
+              .expect(200);
+              should(res.body).have.property('data').Array().length(0);
+          });
+          it('[example 3] show list of 2 new users who share at leats 3 tags with me', async function () {
+            const res = await agent
+              .get('/users?sort=-created&filter[withMyTags]=3&page[offset]=0&page[limit]=2')
+              .set('Content-Type', 'application/vnd.api+json')
+              .auth(loggedUser.username, loggedUser.password)
+              .expect('Content-Type', /^application\/vnd\.api\+json/)
+              .expect(200);
+              should(res.body).have.property('data').Array().length(2);
+
+              const [user1, user2] = res.body.data;
+              testUser(user1, { username: 'user10'});
+              testUser(user2, { username: 'user7'});
+          });
+          it('[example 4] show list of 20 new users who share at leats 1 tags with me', async function () {
+            const res = await agent
+              .get('/users?sort=-created&filter[withMyTags]=1&page[offset]=0&page[limit]=20')
+              .set('Content-Type', 'application/vnd.api+json')
+              .auth(loggedUser.username, loggedUser.password)
+              .expect('Content-Type', /^application\/vnd\.api\+json/)
+              .expect(200);
+              should(res.body).have.property('data').Array().length(6);
+
+            const [ user1, user2, user3, user4, user5, user6 ] = res.body.data;
+              testUser(user1, { username: 'user11'});
+              testUser(user2, { username: 'user10'});
+              testUser(user3, { username: 'user7'});
+              testUser(user4, { username: 'user6'});
+              testUser(user5, { username: 'user3'});
+              testUser(user6, { username: 'user2'});
+          });
+        });
+        context('not logged in', function () {
+          it('error 403', async function () {
+            const res = await agent
+              .get('/users?sort=-created&filter[withMyTags]=2&page[offset]=0&page[limit]=5')
+              .set('Content-Type', 'application/vnd.api+json')
+              .expect('Content-Type', /^application\/vnd\.api\+json/)
+              .expect(403);
+          });
+        });
+      });
+      context('invalid data', function() {
+        it('[invalid \'shareMyTags\' parameter] error 400', async function () {
+          const res = await agent
+            .get('/users?sort=-created&filter[withMyTags]=2&fpage[offset]=0&page[limit]=5')
+            .set('Content-Type', 'application/vnd.api+json')
+            .auth(loggedUser.username, loggedUser.password)
+            .expect('Content-Type', /^application\/vnd\.api\+json/)
+            .expect(400);
+        });
+        it('[invalid \'page.offset\' parameter] parameter is not a number: error 400', async function () {
+          const res = await agent
+            .get('/users?sort=-created&filter[withMyTags]=2&fpage[offset]=text&page[limit]=5')
```

Comment:
Perhaps you have copy-pasted it from the previous query, haven't you? Here the query should fail because of parameter not a number, not because of its invalid name.